mtcpclient/groupofconnections.go

```diff
@@ -46,6 +55,60 @@ func (gc GroupOfConnections) AtLeastOneConnectionInError() bool {
 	return gc.isIn(tcpclient.ConnectionError)
 }
 
-func (gc GroupOfConnections) AtLeastOneConnectionEstablished() bool {
-	return gc.isIn(tcpclient.ConnectionEstablished)
+func (gc GroupOfConnections) AtLeastOneConnectionOK() bool {
+	return gc.isIn(tcpclient.ConnectionEstablished) || gc.isIn(tcpclient.ConnectionClosed)
```

Comment:
Im not sure if here is where we should make the matching of "connection with status X and Y means connection went well", or one step deeper (that may have extra sense, as individual connection status is closer to the logic of tcpclient), so this method should just iterate asking a connection if it was OK.. Current isIn method maybe should be renamed, in any case (rather than "isIn", "containsAConnectionWithStatus(status)" may have a better meaning - apol, it was me who introduced this ;) )